releasenotes/notes/control-flow-depth-size-b598a4eb9d8888eb.yaml

```diff
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    The circuit method :meth:`.QuantumCircuit.depth`, and the :class:`.DAGCircuit`
+    methods :meth:`~.DAGCircuit.depth` and :meth:`~.DAGCircuit.size` gained a
+    ``recurse`` keyword argument for use with control flow.  By default this is
+    ``False`` and will raise an error if control-flow is present, to avoid
+    nonsensical results.  If set to ``True``, a hint to the true value is returned,
+    even though the exact number is generally unknowable.  See each method's
+    documentation for how each control-flow operation affects the output.
+  - |
+    :meth:`.DAGCircuit.count_ops` gained a ``recurse`` keyword argument for
+    recursing into control-flow blocks.  By default this is ``True``, and all
+    operations in all blocks will be returned, as well as the control-flow
+    operations themselves.
```

Comment:
Can you add a bugfix note about previously incorrectly counting `ControlFlowOps`?

Reply:
Done in 61380dd9e.